gfootball/env/observation_processor.py

```diff
@@ -111,11 +158,14 @@ def get_frame(trace):
         field_coords=True,
         color=(0, 255, 0))
     letter = 'H'
+    # Goalkeeper
+    if player_idx == 0:
+      letter = 'G'
     if 'active' in trace and player_idx in trace['active']:
       letter = 'X'
```

Comment:
Why do we still have this 'X'? I thought we always numer players with G, 0, 1,... 9 and in the table we show ID of the controlled player.